_posts/en/newsletters/2025-12-12-newsletter.md

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Bitcoin Optech Newsletter #384'
+permalink: /en/newsletters/2025/12/12/
+name: 2025-12-12-newsletter
+slug: 2025-12-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+FIXME:bitschmidty
+
+## News
+
+- **Critical vulnerabilities fixed in LND 0.19.0:** Matt Morehouse
+  [posted][morehouse delving] to Delving Bitcoin about critical vulnerabilities
+  fixed in LND 0.19.0. In this disclosure, there are 3 vulnerabilities
+  mentioned: one denial of service and two theft-of-funds.
+  - **The Infinite Inbox DoS:** This a denial of service venuralbility that took
+    advantage of LND allowing as many peers as there were available file
+    descriptors, the attacker could then open multiple connections to the victum
+    and spam query_short_channel_ids messages of size 64 KB while keeping the
+    connection open until LND ran out of memory. The mitigation for this
+    vulnerability was implemented in LND 0.19.0 on March 12th, 2025.
+
+  - **The Excessive Failback Exploit #2:** This is the first of the two
```

Comment:
Lowercasing the names of these items. Also rewording to be more descriptive for readers.